kustomize/startMinikubeDeployLocalDev.sh

```diff
@@ -13,10 +13,6 @@ mkdir -p ${KUSTOMIZE_DEPLOY_DIR}/overlays
 cp -r ${KUSTOMIZE_DIR}/base ${KUSTOMIZE_DEPLOY_DIR}
 cp -r ${KUSTOMIZE_DIR}/overlays/minikube ${KUSTOMIZE_DEPLOY_DIR}/overlays
 
-echo "Setup Managed Kafka and Managed Connectors"
-. ${BIN_DIR}/configure.sh kafka managed-connectors
-${BIN_DIR}/kafka-setup.sh
```

Comment:
I think we need to partially keep this setup, please see my PR https://github.com/r00ta/sandbox/pull/10

Reply:
Thank you @jomarko , merged